IQSS/dataverse
# add local_lib mvn repo and primefaces themes, pin base image to Ubuntu 22

**What this PR does / why we need it**:

We are getting build errors.

**Which issue(s) this PR closes**:

- Closes #10718
- Closes #10722

**Special notes for your reviewer**:

The Primefaces error gets triggered first but there is a second error having to do with containers (#10722) that I'm fixing in this PR as well.

I tried added just "all-themes" but "themes-project" seems to be required as well.

We used to have a local_lib repo but we removed it in this PR:

- #9844

**Suggestions on how to test this**:

- Make sure you can build.
- Make sure the UI looks ok? I'm not sure how the themes are used.

**Does this PR introduce a user interface change? If mockups are available, please link/include them here**:

I hope not.

**Is there a release notes update needed for this change?**:

No.

**Additional documentation**:

None.
